<launch>
    <!-- gazebo configs -->
    <arg name="gui" default="true"/>

    <!-- vehicle model and world -->
    <arg name="vehicle" default="iris_depth_camera"/>
    <!-- <arg name="world" default="$(find mavlink_sitl_gazebo)/worlds/empty.world"/> -->
    <arg name="world" default="$(find accel_control)/launch/testworld.world"/>

    <!-- MAVROS configs -->
    <arg name="fcu_url" default="udp://:14540@localhost:14557"/>
	
	<!-- Include the MAVROS node with SITL and Gazebo -->
	<include file="$(find px4)/launch/mavros_posix_sitl.launch">
        <arg name="gui" value="$(arg gui)"/>
        <arg name="fcu_url" value="$(arg fcu_url)"/>
        <arg name="vehicle" value="$(arg vehicle)"/>
        <arg name="world" value="$(arg world)"/>
	</include>
</launch>